### ISSUE TITLE
Module reference in @registermodel

### ISSUE THREAD

**Issue by moesphere** [ClapeyronThermo/Clapeyron.jl#157]
Hello,

I think the first line of the @registermodel macro, i.e., the `@__MODULE__()` should be replaced by `__module__` as it should refer to the module from which the macro is being invoked, and not the module in which the code is defined.

First line in https://github.com/ClapeyronThermo/Clapeyron.jl/blob/4136658d31edbbfde794fbefd588024d9ddd79a2/src/utils/macros.jl#L376-L382:

Replace
```julia
_model = getfield(@__MODULE__(),model)
```
with
```julia
_model = getfield(__module__,model)
```

**Comment by longemen3000** [comment]
yes, it is indeed a bug. I will fix it in #156

**Comment by longemen3000** [comment]
@moesphere fixed in release 0.4.8
